lib/main.js

```diff
@@ -91,6 +97,16 @@ module.exports = {
         }
       }
     }));
+    this.subscriptions.add(atom.config.observe('linter-eslint.localNodeModules', function (localNodeModules) {
+      if (localNodeModules) {
+        atom.config.set('linter-eslint.useGlobalEslint', false);
+      }
+    }));
+    this.subscriptions.add(atom.config.observe('linter-eslint.useGlobalEslint', function (useGlobalEslint) {
+      if (useGlobalEslint) {
+        atom.config.set('linter-eslint.localNodeModules', '');
```

Comment:
Permanently changing the value of the setting based on whether or not the Use Global setting is checked is a bad idea, you should simply not use it in the worker if this is enabled if that was your intent.